cluster/gce/addons/konnectivity-agent/konnectivity-agent-ds.yaml

```diff
@@ -26,11 +26,13 @@ spec:
           effect: "NoExecute"
       nodeSelector:
         kubernetes.io/os: linux
+      hostNetwork: true
       containers:
         - image: k8s.gcr.io/kas-network-proxy/proxy-agent:v0.0.25
           name: konnectivity-agent
           command: ["/proxy-agent"]
           args: [
+                  "--v=5",
```

Comment:
not knowing anything about how the agent logs, V5 sounds pretty high... is this so verbose as to cause problems?